common/determined_common/constants.py

```diff
@@ -29,3 +29,5 @@
 ERROR = "ERROR"
 
 TERMINAL_STATES = {COMPLETED, CANCELED, ERROR}
+
+SHARED_FS_CONTAINER_PATH = "/determined_shared_fs"
```

Comment:
blocking: we may want to leave `SHARED_FS` out of this constant name since its the path to checkpoints inside the container regardless of the persistent storage manager subclass.

Reply:
What do you mean?  I'm pretty sure that S3StorageManager is going to use a randomly-chosen temporary directory.

Reply:
Got it. Then this makes sense.